src/sentry/static/sentry/app/views/performance/landing.tsx

```diff
@@ -178,18 +178,30 @@ class PerformanceLanding extends React.Component<Props, State> {
 
     // This is a temporary change for trends to test adding a default count to increase relevancy
     if (viewKey === FilterViews.TRENDS) {
+      const hasStartAndEnd = newQuery.start && newQuery.end;
+      if (!newQuery.statsPeriod && !hasStartAndEnd) {
+        newQuery.statsPeriod = DEFAULT_TRENDS_STATS_PERIOD;
+      }
       if (!newQuery.query) {
         newQuery.query =
           'count():>1000 transaction.duration:>0 p50():>0 avg(transaction.duration):>0';
```

Comment:
Is it necessary to add `p50()>0` and `avg(transaction.duration):>0` here if `transaction.duration>0` is already present? If not, is it possible to just delete this if statement since the two conditions following this already adds the remaining filters.

Reply:
It's not necessary, I was actually adding them for internal just as an indication of what you can filter on, I'll probably change it back to just `transaction.duration`